### PR TITLE
build: Release v22.0

### PR DESCRIPTION
**Pull request by rbradford**
Signed-off-by: Rob Bradford <robert.bradford@intel.com>

### CODE REVIEW

**Review by liuw — APPROVED** [review]

File: release-notes.md [hunk 1]
```diff
@@ -1,8 +1,19 @@
+- [v22.0](#v220)
+    - [GDB Debug Stub Support](#gdb-debug-stub-support)
+    - [`virtio-iommu` Backed Segments](#virtio-iommu-backed-segments)
+    - [Before Boot Configuration Changes](#before-boot-configuration-changes)
+    - [`virtio-balloon` Free Page Reporting](#virtio-balloon-free-page-reporting)
+    - [Support for Direct Kernel Booting with TDX](#support-for-direct-kernel-booting-with-tdx)
+    - [PMU Support for AArch64](#pmu-support-for-aarch64)
+    - [Documentation Under CC-BY-4.0 Licese](#documentation-under-cc-by-40-licese)
```
Comment (sboeuf) [inline release-notes.md:8]:
```suggestion
    - [Documentation Under CC-BY-4.0 License](#documentation-under-cc-by-40-license)
```

File: release-notes.md [hunk 2]
```diff
@@ -183,18 +194,108 @@
...
+directly boot into the kernel with TDX. The [TDX
+documentation](https://github.com/cloud-hypervisor/cloud-hypervisor/blob/main/docs/intel_tdx.md#tdshim)
+has been updated for this usage.
+
+### PMU Support for AArch64
+
+A PMU is now available on AArch64 for guest performance profiling. This will be
+exposed automatically if available from the host.
+
+### Documentation Under CC-BY-4.0 Licese
```
Comment (sboeuf) [inline release-notes.md:241]:
```suggestion
### Documentation Under CC-BY-4.0 License
```

Comment (rbradford) [reply release-notes.md:241]: I did manage to spot & fix it in the version that went in the release notes.

Comment (sboeuf) [reply release-notes.md:241]: Cool!

**Pull request merged**